diesel/src/sqlite/connection/mod.rs

```diff
@@ -107,6 +107,10 @@ impl Connection for SqliteConnection {
     fn get_transaction_depth(&self) -> i32 {
         self.transaction_depth.get()
     }
+
+    fn setup_helper_functions(&self) {
+        // this will be implemented at least when timestamps are supported in SQLite
```

Comment:
Does this need to be on connection given that it's PG specific right now? Should this at least have `unimplemented!()`?